README.md

```diff
@@ -37,6 +37,17 @@ to your Github account:
 
 You should now receive an email whenever the changelog is updated.
 
+---
+### 22 June 2023
+The following changes were deployed into sandbox and production:
```

Comment:
Have these actually been released i.e. made available in production?

Reply:
@mattstephens-hmrc Yes they have been deployed to both ET and Production